virt/virt-2-5-release-notes.adoc

```diff
@@ -131,6 +131,8 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 
 * When a node fails in user-provisioned installations of {product-title} on bare metal deployments, the virtual machine does not automatically restart on another node. Automatic restart is supported only for installer-provisioned installations that have machine health checks enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886437[*BZ#1886437*])
 
+* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider. link:https://bugzilla.redhat.com/show_bug.cgi?id=1893160[(*BZ#1893160*)]
```

Comment:
Customers are not interested in which doc bug this PR has fixed. We should point them to where they check what is broken and when it is expected to be fixed - in this case https://bugzilla.redhat.com/show_bug.cgi?id=1887456

As a rule of thumb, no doc bugs should be mentioned in the docs, only product bugs which are expected to remain open when the docs go live.

Reply:
Yes, thank you. I have changed the link to point to the product bug.